docs/data/sdks/go.md

```diff
@@ -0,0 +1,404 @@
+---
+title: Go SDK (Alpha)
+description: The Amplitude Go SDK installation and quick start guide.
+icon: fontawesome/brands/golang
+---
+
+!!!info "SDK Resources"
+    - [Go SDK Repository :material-github:](https://github.com/amplitude/Amplitude-Go)
+    - [Go SDK Releases :material-code-tags-check:](https://github.com/amplitude/Amplitude-Go/releases)
+
+--8<-- "includes/no-ampli.md"
+
+The Go SDK lets you send events to Amplitude. This library is open-source, check it out on [GitHub](https://github.com/amplitude/Amplitude-Go).
+
+## Getting Started
+
+### Installation
+
+Install `Amplitude-Go` for analytics:
+
+```bash
+go get https://github.com/amplitude/Amplitude-Go
+```
+
+## Usage
+
+### Initializing SDK
+
+Initialization is necessary before any instrumentation is done. The API key for your Amplitude project is required to construct a Config struct. Use that Config struct to initialize a client struct which implements Client interface. It can be used across requests after it is initialized.
+
+```GO
+
+import "github.com/amplitude/Amplitude-Go/amplitude"
```

Comment:
replace name to the new name

Reply:
replace it with github.com/amplitude/analytics-go